nerfstudio/model_components/exporters.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2022 The Nerfstudio Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Algorithms for exporting data."""
+
+from __future__ import annotations
+
+import sys
+from typing import Tuple
+
+import numpy as np
+import open3d as o3d
+import torch
+from rich.console import Console
+from rich.progress import (
+    BarColumn,
+    Progress,
+    TaskProgressColumn,
+    TextColumn,
+    TimeRemainingColumn,
+)
+
+from nerfstudio.configs.base_config import Config  # pylint: disable=unused-import
+from nerfstudio.pipelines.base_pipeline import Pipeline
+
+CONSOLE = Console(width=120)
+
+
+def generate_point_cloud(
+    pipeline: Pipeline,
+    num_points: int = 1000000,
```

Comment:
1e6

Reply:
1e6 is a float, not an int. I can add the typecast, but at that point it seems a bit excessive.